templates/CRM/Custom/Form/ContactReference.tpl

```diff
@@ -28,7 +28,7 @@
 {literal}
 <script type="text/javascript">
   CRM.$(function($) {
-    var $field = $("{/literal}#{$element_name|replace:']':''|replace:'[':'_'}{literal}");
+    var $field = $("{/literal}#{if $prefix}{$prefix}_{/if}{$element_name|replace:']':''|replace:'[':'_'}{literal}");
```

Comment:
I think we are better to half any _ in the prefix - otherwise it's too easy to have in some places & not others